### DISCUSSION TITLE
Niko touchscreen (550-20102) for home-assistant kiosk?

### DISCUSSION THREAD

**Discussion by davidvf** [joleys/niko-home-control-II#119] [Q&A]
Hello, this is not really an issue regarding this integration for home-assistant.
However, since you are all home automation enthusiasts with both home-assistant and NHC II, i think this might be a good place to ask.

So i have the niko touchscreen in my living room and to be honest, i barely ever use it.
With this integration in HASS, it is much faster to pick up my phone and control the lights from HASS.
I am wondering if anyone ever tried to reimage the niko touchscreen with Lineage OS for example? That would allow us to put up home-assistant on the touchscreen in kiosk mode. This would open up a lot of opportunities, so i can have access to all of my HASS integrations instead of only NHC II on the touchscreen.

**Comment by Pitsaer** [comment]
Personally, I wouldn't do it because I use the screen as a back-up for a BL 550-22001 door station. And that's what Niko recommends, to have a screen wired for access control. 

But like you, I prefer H.A with NHC2 integration to the official Niko screen. I even unplugged my Digital black 550-18001... 
But installing H.A on the 550-18001 would be a great idea ;-)
